src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs

```diff
@@ -66,13 +66,14 @@ public class CSharpFunctionInvoker : ScriptFunctionInvokerBase
         private static string GetTriggerInputName(FunctionMetadata functionMetadata)
         {
             BindingMetadata triggerBinding = functionMetadata.Bindings.FirstOrDefault(b => b.IsTrigger);
-            
+
+            string triggerName = null;
             if (triggerBinding != null)
             {
-                return triggerBinding.Name;
+                triggerName = triggerBinding.Name;
```

Comment:
I see, so the difference is that `triggerBinding.Name` could be null. Looks good.